### ISSUE TITLE
Move data option not working with set torrent location

### ISSUE THREAD

**Issue by ginja85** [jfurrow/flood#581]
When setting the location of a torrent there is a checkbox "Move data" which should take existing downloaded data and move it to the new location.

## Expected Behavior
Existing torrent data should be moved to new location.

## Current Behavior
Data is left in previous location and torrent data is re-downloaded at new location.

## Possible Solution
My initial thought is that the move data option is not correctly wired up to do it's intended function.

## Steps to Reproduce (for bugs)
1. Add torrent via flood interface, wait until torrent completes download
2. Right click torrent and select set torrent location
3. In the following dialog choose a new folder location for the torrent
4. Observe torrent will re-download and data is held at old location

## Context
The torrent in question has multiple files under a folder structure e.g.
```
  - This.Is.A.Torrent.Folder.Containing.Files
  | -- FileA.txt
  | -- FileB.exe
  | -- FileC.png
```

On initial adding and download I discover file A, B and C do not appear to be contained in folder "This.Is.A.Torrent.Folder.Containing.Files" I attempt to set the torrent location again in order to fix this and where this issue then occurs. The problem with a torrent being added and flood ignoring that the torrent has a root folder may be another separate issue to this. Possible torrent location not being set correctly on intial add?

## Your Environment
* Version used:
  + Version (stable release) `v1.0.0`
  + Commit ID (development release) `2fd0edae15d21bcb79421b6ca1b07c9ff7705bc7`
* Environment name and version: Chrome 64 Node v9
* Operating System and version: Debian 4.9.30

**Comment by munkyboy** [comment]
I had this same problem. I'm using rtorrent and flood in different docker containers. When I have each container use the same filesystem, the move functionality works as expected. Hope that helps you.

**Comment by bootldrDNB** [comment]
This is still an issue, running rTorrent and flood in different docker containers here as well.

**Comment by jfurrow** [comment]
Yeah, the Flood server currently expects the exact same filesystem shared between the Flood server instance and the rTorrent instance. This is doable with Docker instances — just provide the same filesystem mount point path for each instance.

So this isn't a bug, this is a feature request for translating paths from the Flood server instance's accessible filesystem to the rtorrent instance's accessible filesystem.
